2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -769,91 +750,109 @@ Documenti è organizzata in modo da suddividere la documentazione necessaria all
+
+====== Backlog
+Ogni task da svolgere è segnato all'interno del backlog di Jira. Durante la pianificazione dello sprint, si definisce lo sprint backlog, il sottoinsieme di attività provenienti dal backlog che si intendono portare a termine entro la conclusione dello sprint.
+A differenza dello sprint backlog definito durante la pianificazione, il backlog viene espanso man mano che si riscontrano nuovi task necessari o a seguito di decisioni prese durante le riunioni interne o esterne.
+
+====== Board
+Le board di Jira permettono, similmente allo sprint backlog, di avere una visione d'insieme delle attività da svolgere, ma con un approccio più visuale e intuitivo.
+I task sono organizzate in quattro colonne, rappresentanti lo stato:
+- *da completare*: non ancora avviati, ovvero non esiste il branch dedicato;
+- *in corso*: in fase di svolgimento, ovvero branch dedicato al task creato;
+- *in verifica*: in fase di review, dopo l'apertura di una pull request;
+- *completato*: task concluso, ovvero branch dedicato chiuso a seguito di merge sul ramo principale.
+
+====== Timeline
+La timeline di Jira permette di avere una visione delle attività incentrata sulle tempistiche e le relazione tra i task.
+Permette inoltre di mostrare il grafico di Gantt delle attività evidenziando i rapporti di dipendenza tra i task e stabilendo le scadenze necessarie per il loro svolgimento.
+
+====== Grafici
+Jira offre la possibilità di produrre grafici e report relativi all'avanzamento e alla tracciabilità dei task. Tali strumenti permettono di avere delle metriche di valutazione dell'andamento del progetto e di individuare eventuali criticità.
+Il gruppo utilizza come metrica principale il burndown chart, che permette di avere una visione dell'avanzamento delle attività in base al tempo, basato sugli story point di ogni attività.
+
+
+==== Controllo delle release
+Il controllo delle release viene gestito tramite il meccanismo di pull request di GitHub. 
+Prima di integrare i nuovi cambiamenti, viene aperta una pull request dall'assegnatario del task. La pull request deve avere un titolo valido (come descritto nel paragrafo dedicato a GitHub Actions) e deve essere designato almeno un reviewer. Di norma il reviewer di base è il Verificatore, che svolge una supervisione sulla correttezza sintattica e semantica dei contenuti.
+Il Verificatore può essere affiancato da altre figure in caso, per quanto riguarda la semantica e il contenuto del materiale da revisionare, ci sia bisogno di una figura con delle competenze specifiche. Per ogni richiesta di modifiche da apportare vengono aperte delle conversation, in cui è possibile evidenziare le linee del documento che hanno bisogno di cambiamenti, oltre a stabilire un canale di comunicazione fra assegnatario e Verificatore.
```

Comment:
girerei la prima frase in "Nel caso in cui ci sia bisogno di una figura con delle competenze specifiche per quanto riguarda la semantica e il contenuto del materiale da revisionare, il Verificatore può essere affiancato da altri membri del gruppo."